website/www/site/content/en/documentation/sdks/python-machine-learning.md

```diff
@@ -233,6 +234,16 @@ For more information, see the [`PredictionResult` documentation](https://github.
 For detailed instructions explaining how to build and run a pipeline that uses ML models, see the
 [Example RunInference API pipelines](https://github.com/apache/beam/tree/master/sdks/python/apache_beam/examples/inference) on GitHub.
 
+## Side Inputs to Update Models
+From Beam 2.45.0, the RunInference PTransform will accept a side input of `ModelMetadata`, which is a `NamedTuple` containing the `model_id` and `model_name`.
```

Comment:
```suggestion
From Beam 2.46.0, the RunInference PTransform will accept a side input of `ModelMetadata`, which is a `NamedTuple` containing the `model_id` and `model_name`.
```

2.45 has been cut already unfortunately.

Also, could we split website changes into a separate PR. Ideally we would merge those once the feature becomes available to avoid confusing people.